resources/helm/v2.1/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -182,24 +181,6 @@ spec:
             value: {{ .Values.global.tls.ecdhCurves }}
 {{- end }}
 {{- end }}
-{{- if .Values.global.tls }}
```

Comment:
don't we still need this?

Reply:
Yes, but it's moved to the charts in maistra/istio, so this is just removing a duplicate section (look above this line).